<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>digitOrderEditor</class>
 <widget class="QDialog" name="digitOrderEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digit Order Editor</string>
  </property>
  <property name="whatsThis">
   <string>Digit Order Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="sidebarLayout">
     <item>
      <widget class="QLabel" name="posiLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Position</string>
       </property>
       <property name="whatsThis">
        <string>Position</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="charLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Character</string>
       </property>
       <property name="whatsThis">
        <string>Character</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reprLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Representation</string>
       </property>
       <property name="whatsThis">
        <string>Representation</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shuffleButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Randomizes the digit order</string>
       </property>
       <property name="whatsThis">
        <string>Randomizes the digit order</string>
       </property>
       <property name="text">
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets the digit order</string>
       </property>
       <property name="whatsThis">
        <string>Resets the digit order</string>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vt_spc_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Saves the digit order and closes</string>
       </property>
       <property name="whatsThis">
        <string>Saves the digit order and closes</string>
       </property>
       <property name="text">
        <string>Save &amp;&amp;
Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vt_spc_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="devButtonsLayout">
       <item>
        <layout class="QHBoxLayout" name="devlayout_hz_1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QSpinBox" name="numCol_input">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Number of columns</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Sets the number of columns in character display table</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="itemSize_input">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Item size</string>
           </property>
           <property name="whatsThis">
            <string>Sets the item size in character display table</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="updateTableButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Update the character display table</string>
         </property>
         <property name="whatsThis">
          <string>Updates the character display table</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxChar_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="toolTip">
          <string>Maximum number of characters</string>
         </property>
         <property name="whatsThis">
          <string>Sets the maximum number of characters to display</string>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1114111</number>
         </property>
         <property name="value">
          <number>1114111</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="charTableLayout">
     <item>
      <layout class="QHBoxLayout" name="charButtonsLayout">
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="toolTip">
          <string>Export the digit order for external rearranging</string>
         </property>
         <property name="whatsThis">
          <string>Exports the digit order to a file</string>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>infomation label</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="toolTip">
          <string>Cancel file import</string>
         </property>
         <property name="whatsThis">
          <string>Cancels file import</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="toolTip">
          <string>Import custom digit order</string>
         </property>
         <property name="whatsThis">
          <string>Imports a digit order from a file</string>
         </property>
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
